### ISSUE TITLE
[PNI Editorial Updates] Redoing PNI User Test

### ISSUE THREAD

**Issue by beccaklam** [mozilla/foundation.mozilla.org#9458]
The user test ran in #9127 struggled with not enough real content in the prototype and perhaps not enough context setting/understanding of what a prototype is. So, we are redoing the test and this time with a staging link! 🎉

Goals
- [x] Prep [user testing script](https://docs.google.com/document/d/1YcvLH8mEQlnwPLIw_yFUkrBpFEAerOGC38vcuCyR5yQ/edit) to match [staging link](https://foundation-pni-2022.herokuapp.com/en/privacynotincluded/)
- [x] Check in to see if all the latest updates have been merged onto the staging branch
- [x] Set up test in UserTesting.com
- [x] Run test
- [x] Watch videos
- [x] Take notes on feedback on how to improve the site/updates

**Comment by beccaklam** [comment]
The script has been prepped, just waiting for some PRs to land. @fessehaye Do you know when all the PRs will be merged?

**Comment by beccaklam** [comment]
Added more contributors to the test today, and ran a new test to include UK and German contributors. I took notes on the videos in usertesting.com and started a [google doc to track trends](https://docs.google.com/spreadsheets/d/1UCwj8xF8rs_3mOaNn51F8SnHGLV6duo5H9LYXx2iWh8/edit?usp=sharing). I'll add six more contributors tomorrow (three from the UK, three from Germany) to try to solidify the trends more.
